packages/dds/tree/src/test/tableSchema.spec.ts

```diff
@@ -805,169 +805,162 @@ describe("TableFactory unit tests", () => {
 			});
 		});
 
```

Comment:
Note for reviewers: I highly recommend enabling "hide whitespace" when reviewing this PR.